pkg/etc/config_test.go

```diff
@@ -67,8 +69,8 @@ func TestGetConfig(t *testing.T) {
 					IdleTimeout:    parseDuration(t, "1h2m3s"),
 				},
 				AquaCSP: AquaCSP{
-					User:        "",
-					Password:    "",
+					User:        "scanner",
```

Comment:
nit: you might want to change the field name to Username to match the names you've used elsewhere. Not a big deal

Reply:
Done!